commands/deploy.go

```diff
@@ -261,12 +268,18 @@ func deployImage(
 	}
 
 	functionResourceRequest1 := proxy.FunctionResourceRequest{}
-	proxy.DeployFunction(fprocess, gateway, functionName,
+	statusCode := proxy.DeployFunction(fprocess, gateway, functionName,
 		image, registryAuth, language,
 		deployFlags.replace, envvars, network,
 		deployFlags.constraints, deployFlags.update, deployFlags.secrets,
 		labelMap, functionResourceRequest1)
 
+	functionsStatusCodes[functionName] = statusCode
```

Comment:
Why store the result in the map if it could potentially be a valid status?

Reply:
Because i need to give it as an argument to the function. I messed the commit before that for some reason.